yeonjae/programmers-1/stack-queue/같은숫자는싫어.js

```diff
@@ -1 +1,20 @@
-function solution(arr) {}
+function solution(arr) {
+  const resultArr = [];
+
+  for(let i = 0; i < arr.length; i++) {
+    if(arr[i] !== arr[i+1]) {
+      resultArr.push(arr[i]);
+    }
+  }
+  return resultArr
+}
```

Comment:
```suggestion
function solution(arr) {
  const result = [];

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] !== arr[i + 1]) {
      resultArr.push(arr[i]);
    }
  }
  return result;
}
```
나보다 훨씬 깔끔하게 풀었고 컨벤션이랑 변수명만 수정했습니다 :)

Reply:
감사합니다 :)